devices/motorola-potter/default.nix

```
@@ -0,0 +1,61 @@
+{ config, lib, pkgs, ... }:
+
+{
+  mobile.device.name = "motorola-potter";
+  mobile.device.identity = {
+    name = "Moto G5 Plus";
+    manufacturer = "Motorola";
+  };
+
+  mobile.hardware = {
+    soc = "qualcomm-msm8953";
+    ram = 1024 * 3;
```

```suggestion
    ram = 1024 * 2;
```

According to [this page](https://www.gsmarena.com/motorola_moto_g5_plus-8453.php), the "Moto G5 Plus" exists in variants with 2, 3 and 4 GiB of RAM.

This documents the smallest available RAM size, but is currently totally unused.

I *assume* it might be used soon for the installer, to provide hints at build time to the user that the "copytoram" installer variants may not work.

Otherwise setting a lower amount than your actual device has *has no impact*.